examples/advanced/vertical_xgboost/README.md

```diff
@@ -47,7 +47,13 @@ Next, we can use `FedXGBHistogramExecutor` and set XGBoost training parameters i
 
 Lastly, we must subclass `XGBDataLoader` and implement the `load_data()` method. For vertical federated learning, it is important when creating the `xgb.Dmatrix` to set `data_split_mode=1` for column mode, and to specify the presence of a label column `?format=csv&label_column=0` for the csv file. To support PSI, the dataloader can also read in the dataset based on the calculated intersection, and split the data into training and validation.
 
-> **_NOTE:_** For secure mode, make sure to provide the required certificates for the federated communicator. Also as of now, GPUs are not yet supported by vertical federated XGBoost.
+> **_NOTE:_** For secure mode, make sure to provide the required certificates for the federated communicator.
+
+### GPU Support
+By default, CPU based training is used.
+
+In order to enable GPU accelerated training, first ensure that your machine has CUDA installed and has at least two GPUs to simulate two clients.
+Then in `config_fed_client.json` set `"use_gpus": true` and  `"tree_method": "hist"` in `xgb_params`.
```

Comment:
Do we really need 2 GPUs for simulating 2 clients?

Can we use 1 GPU to run 2 clients' training (parallel or one by one)?

Reply:
yep, we need 2 GPUs with nccl, a single GPU will not work [https://github.com/dmlc/xgboost/pull/9552](ref)